mne_icalabel/megnet/_utils.py

```diff
@@ -0,0 +1,44 @@
+# %%
+import numpy as np
+
+
+# Conversion functions
+def cart2sph(x, y, z):
+    xy = np.sqrt(x * x + y * y)
+    r = np.sqrt(x * x + y * y + z * z)
+    theta = np.arctan2(y, x)
+    phi = np.arctan2(z, xy)
+    return r, theta, phi
+
+
+def pol2cart(rho, phi):
+    x = rho * np.cos(phi)
+    y = rho * np.sin(phi)
+    return x, y
```

Comment:
These functions are also defined for ICLabel, so I wonder if we can pull them out for a general utility functions related to geometry.

https://github.com/mne-tools/mne-icalabel/blob/b2fc448dd2a5718eeeefee14d051525f2cb4b209/mne_icalabel/iclabel/_utils.py#L97

Reply:
`pol2cart` is indeed a duplicate, `cart2sph` returns the element in a different order and it's a bit annoying to change the order. We can keep code de-duplication for a future PR.